src/Form/Type/Organization/ChangeAnonymousAccessType.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Buddy\Repman\Form\Type\Organization;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+class ChangeAnonymousAccessType extends AbstractType
+{
+    public function getBlockPrefix(): string
+    {
+        return '';
+    }
+
+    /**
+     * @param array<mixed> $options
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('hasAnonymousAccess', CheckboxType::class, [
+                'label' => 'Allow anonymous users',
+                'required' => false,
```

Comment:
please add a "hint" that enabling anonymous access allows to browse and download all organization packages

Reply:
<img width="784" alt="Screenshot 2020-06-19 at 11 20 26" src="https://user-images.githubusercontent.com/3144291/85117264-e3460000-b21e-11ea-8c9c-63fe5c94aedb.png">